app/src/main/res/values-v21/styles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="calculatorButton">
+        <item name="android:textColor">@color/green</item>
+        <item name="android:textSize">@dimen/main_btn_size</item>
+        <item name="android:backgroundTint">@color/black</item>
+        <item name="android:padding">@dimen/main_btn_padding</item>
+    </style>
+
+    <style name="calculatorScreen">
+        <item name="android:background">@color/dark_green</item>
+        <item name="android:textSize">@dimen/main_result_tv_size</item>
+        <item name="android:text">@dimen/main_result_tv_sample</item>
```

Comment:
android:text should get a string, not a dimen

Reply:
Done